core/src/main/scala/java/text/SimpleDateFormat.scala

```diff
@@ -10,6 +10,67 @@ class SimpleDateFormat(private[this] val pattern: String,
 
   //def this() = this("???", DecimalFormatSymbols.getInstance())
 
+  def format(date: Date): String = {
```

Comment:
Well I see that you are only supporting the numeric formatting. If so then this doesn't matter but if we want to do afull implementation we'd need to use `symbols`

This is very little used and if nobody wants to go to the trouble to fully implement I think I'm ok with this